client/src/game/physics.zig

```diff
@@ -0,0 +1,92 @@
+const rl = @import("raylib");
+const rm = rl.math;
+const GameState = @import("../game/state.zig");
+const std = @import("std");
+
+const heightAxis: rl.Vector3 = .{
+    .x = 0,
+    .y = 1,
+    .z = 0,
+};
+
+pub const Character = struct {
```

Comment:
This is just a namespace, not a struct meant for instantiation. Don't use an uppercase name.